readme.md

```diff
@@ -180,7 +185,11 @@ customer or a charge, but only used once.
 	var tokenService = new StripeTokenService();
 	StripeToken stripeToken = tokenService.Create(myToken);
 
-Tokens are very useful if you don't want to store the customers credit card information on your servers (pci compliance).
+You should never create tokens with Stripe.net with real credit cards, since
+giving your server access to unredacted card data greatly increases your
+PCI-DSS compliance burden.
```

Comment:
I'm fine with getting rid of the "never" wording. However, I'd lean towards targeting this documentation at users that aren't familiar with PCI-DSS, and try to steer them away from common pitfalls. My assumption is that if it's actually your intention to do a SAQ-D, you're fairly familiar with how this stuff works, and need less guidance on how to set up your system in a compliant way.

(As an interesting microcosm of this problem, until recently, the top google result for "stripe.net tutorial" suggested creating tokens through stripe.net. An edit was later made to the tutorial warning readers about the compliance implications of this, and suggesting stripe.js/checkout. https://cmatskas.com/processing-payments-on-your-site-using-stripe-and-net/)

I'm leaning towards some wording along the lines of "from a security / compliance perspective, server-side tokenization has almost no advantages over directly passing unredacted card details to the API."